modules/rendering.py

```diff
@@ -126,22 +126,29 @@ def __render_rays_test(
         xyzs = ray_o_local + ts[:, None] * ray_d_local
         dirs = ray_d_local
 
-        sigmas, rgbs = model(xyzs, dirs)
+        # TODO: model forward
```

Comment:
Could you make this part guarded by a Plenoxel option? so that it won' t break the original NGP